js/page.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+(function () {
+  var ADVERTISEMENT_AMOUNT = 8;
+
+  var activatePage = function () {
+    var adverts = window.generateAdvers(ADVERTISEMENT_AMOUNT);
```

Comment:
```suggestion
    var adverts = [];
```

Например, если без имитации асинхронности просто задать пустой массив, то страница должна переходить в активное состояние при первом взаимодействие с главной меткой, но _форма фильтров_ будет заблокирована.

См. [`ТЗ 5.9`](https://up.htmlacademy.ru/javascript/19/project/keksobooking#keksobooking-5-9):
> Форма, с помощью которой производится фильтрация похожих объявлений на момент открытия страницы заблокирована и **становится доступной только после окончания загрузки всех похожих объявлений**

См. внимательно [прошлое замечание](https://github.com/htmlacademy-javascript/987137-keksobooking-19/pull/4#discussion_r386149060).